internal/webstomp/webstomp.go

```diff
@@ -86,6 +86,7 @@ func (c *Client) Consume(consumer messageConsumer) {
 				c.logger.Infof("failed to aknowledge message, err: %v\n", err)
```

Comment:
It was there before:
```suggestion
				c.logger.Infof("failed to acknowledge message, err: %v\n", err)
```